live-ingester/src/test/java/org/alfresco/hxi_connector/live_ingester/util/ContainerSupport.java

```diff
@@ -64,16 +66,26 @@ public class ContainerSupport
     public static final String REPO_EVENT_TOPIC = "repo.event.topic";
     public static final String ATS_QUEUE = "ats.queue";
     public static final String REQUEST_ID_PLACEHOLDER = "_REQUEST_ID_";
-
     private static ContainerSupport instance;
-
     private Session session;
     private MessageProducer repoEventProducer;
     private MessageConsumer atsConsumer;
 
+    public static final String ATS_RESPONSE_QUEUE = "ats.response.queue";
+    private MessageProducer atsEventProducer;
+    public static final String SFS_ENDPOINT = "/alfresco/api/-default-/private/sfs/versions/1/file/e71dd823-82c7-477c-8490-04cb0e826e66";
+    private static final int OK_SUCCESS_CODE = 200;
+    private static final String HX_INSIGHT_PRE_SIGNED_URL_PATH = "/pre-signed-url";
+    private static final String HX_INSIGHT_LOCATION_PATH = "/ingestion-base-path";
+    private static final String HX_INSIGHT_RESPONSE_BODY_PATTERN = "{\"%s\": \"%s\"}";
+    static final String STORAGE_LOCATION_PROPERTY = "preSignedUrl";
+    private static final String OBJECT_KEY = "dummy-file.pdf";
+    private static final String OBJECT_CONTENT_TYPE = "application/pdf";
+    LocalStorageClient s3StorageMock;
```

Comment:
please move all "static" fields up and leave all non-static on the bottom

Reply:
and also I would make `LocalStorageClient s3StorageMock;` private and rename it to something like "localS3StorageClient" or "localStorageClient" as now it indicates that this is a mock of some class which can be misleading

Reply:
updated in [22a68e1](https://github.com/Alfresco/hxinsight-connector/pull/142/commits/22a68e10c384f2f12b28bf6864bc9963fd6b39f1)